EIPS/eip-7862.md

```diff
@@ -0,0 +1,149 @@
+---
+eip: 7862
+title: Delayed Execution Layer State Root
+description: Delays the ExecutionPayload's state root reference by one block.
+author: Charlie Noyes <charlie@paradigm.xyz>, Dan Robinson <dan@paradigm.xyz>, Justin Drake <justin@ethereum.org>, Toni Wahrstätter (@nerolation)
+discussions-to: https://ethereum-magicians.org/t/eip-7862-delayed-execution-layer-state-root/22559
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-12-23
+---
+
+## Abstract
+
+This EIP proposes that, for each block `n`, the `state_root` in block `n`'s `ExecutionPayload` references the **post-state of block `n-1`**. As a result, block `n` no longer needs to include or verify the newly computed state root for block `n`’s own transactions. This removes the expensive state root computation from the block-propagation critical path, reducing end-to-end latency for block production and potentially increasing throughput.
+
+## Motivation
+
+Currently, every Ethereum block includes two `state_root`. 
```

Comment:
I find the wording of this statement incorrect, EL blocks have only one state root, and this is the state root of Ethereum's "world state" (MPT of accounts)

Reply:
the EIP is talking about the full block as it references `ExecutionPayload`